pageserver/src/tenant/mgr.rs

```diff
@@ -336,10 +336,20 @@ pub async fn delete_timeline(
     Ok(())
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum TenantStateError {
+    #[error("tenant is not found")]
```

Comment:
That would not print `TenantId`, afaik, you'd need `{0}` formatter added? Or this was not the goal?